doc/api-examples/ruby/list_nodes.rb

```diff
@@ -0,0 +1,15 @@
+require "dbus"require "dbus"
+sysbus = DBus.system_bus
+
+bc_service = sysbus["org.eclipse.bluechi"]
+bc_object = bc_service["/org/eclipse/bluechi"]
+bc_interface = bc_object["org.eclipse.bluechi.Manager"]
+nodes = bc_interface.ListNodes
+
+puts "BlueChi nodes:"
+puts "================"
+nodes.each do |node|
+  print "Name: ", node[0], "\n"
+  print "Path: ", node[2], "\n"
+  print "Status: ", node[1], "\n\n"
```

Comment:
```suggestion
  print "Path: ", node[1], "\n"
  print "Status: ", node[2], "\n\n"
```
These two should be switched - based on the [API description](https://github.com/containers/bluechi/blob/main/data/org.eclipse.bluechi.Manager.xml#L9), the path (`o`) is at position 1.

Reply:
you meant, change lines? like line 13 will be status and 14 Path? or change the position in the array ? We are using a library in ruby, not sure if I can do that without changing the library?

Reply:
Like it is in the diff: The node path should be at array position 1, the status at position 2.